checkST2110.js

```diff
@@ -492,11 +492,11 @@ const extractMTParams = (sdp, params = {}) => {
       if (!fmtpPattern.test(lines[x])) {
         continue;
       }
-      let params = lines[x].split(/a=fmtp:\d+\s+/)[1];
-      // let paramsMatch = params.matchAll(fmtpParams);
+      let fmtParams = lines[x].split(/a=fmtp:\d+\s+/)[1];
+      // let paramsMatch = fmtParams.matchAll(fmtpParams);
```

Comment:
Do you want to keep the comment here?

Reply:
Removed; the exact rationale for using the capture groups via repeated `exec` rather than using a single `matchAll` (in https://github.com/AMWA-TV/sdpoker/commit/7cf971c0c7bb8468ebc62f66ae8a75f4914cf34b) escapes me after 2½ years 😄